src/components/RecipeSearchPage.tsx

```diff
@@ -19,16 +16,20 @@ export function RecipeSearchPage() {
       console.log(searchResults);
     });
   }
+
   return (
     <>
     <SearchBar label="Search for recipe" onSubmit={handleRecipeSearch} />
-    {searchResults?.results?.map((recipe) => {
-        return (<div className="App"><h2>{recipe.title}</h2><img
-        src={recipe.image}
-        alt="Recipe" />
-        <br />
-        </div>);
-      })}
+      <Grid container>
+      {searchResults?.results?.map((item) => {
+
+        return (
+          <Grid item xs={4} sm={4} md={4}>
+            <SearchResultCard title={item.title} description={item.id} image={item.image} 
+            />
+          </Grid>
+        );})}
```

Comment:
When you're using a fat arrow function, you don't need to specify an explicit return statement - so unless you have other code you want to write inside this loop, you'll want to just return the component like this:
```suggestion
      {searchResults?.results?.map((item) => (
          <Grid item xs={4} sm={4} md={4} key={item.id}>
            <SearchResultCard
              title={item.title}
              description={item.id}
              image={item.image} 
            />
          </Grid>
        )}
```
also, make sure to add the key prop with a unique id whenever you're rendering items using an array this.